server/db/models/EventType.js

```diff
@@ -25,6 +25,9 @@ const schema = new Schema({
   members: {
     type: [User],
   },
+  active: {
+    type: Boolean,
```

Comment:
I think it would make more sense for user experience if the default value for this is true

Reply:
Sure, I did that originally but removed it.